lib/build_targets/package.dart

```diff
@@ -178,6 +178,24 @@ class DotnetTpk extends TizenPackage {
         'Install the latest .NET SDK from: https://dotnet.microsoft.com/download',
       );
     }
+
+    // TODO(jsuya): Due to a bug in the `tz set` command, it fails
+    // to automatically generate tizen_dotnet_project.yaml. So
+    // create it manually and `tz set` command. This will be fixed
+    // in the next version of Tizen SDK.
+    // https://github.com/flutter-tizen/flutter-tizen/issues/610
+    if (!tizenProject.hostAppRoot
+        .childFile('tizen_dotnet_project.yaml')
+        .existsSync()) {
```

Comment:
Simply `createSync(recursive: true)` without the `if` statement.

`touch` is not a valid command on Windows.

Reply:
Thank you for your advice.